frappe/integrations/doctype/webhook/webhook.py

```diff
@@ -74,6 +75,13 @@ def validate_repeating_fields(self):
 		if len(webhook_data) != len(set(webhook_data)):
 			frappe.throw(_("Same Field is entered more than once"))
 
+	def validate_secret(self):
+		if self.enable_security and self.webhook_secret:
+			try:
+				self.get_password("webhook_secret", False).encode("utf8")
+			except Exception:
+				frappe.throw(_("Invalid Webhook Secret"))
```

Comment:
passing `False` for `raise_exception` and then trying to catch it?

Reply:
By default it is True
But, the error message coming from get_password is not user friendly.

We can have one more parameter error_msg to send our message or we can do like this